tests/unit/style/custom_properties.rs

```diff
@@ -24,7 +24,7 @@ use style::stylesheets::container_rule::ContainerSizeQuery;
 use style::stylesheets::layer_rule::LayerOrder;
```

Comment:
Not directly relevant to this PR, but these tests (the whole of `tests/unit/style`) exist upstream. We should start syncing (and running) them in the Stylo repo, and then delete them here. Then we'd get these fixes automatically.